### PR TITLE
feat(i18n): add pt-BR support

### PR DESCRIPTION
**Pull request by Kyo-70**
Here's my contribution to adding my Brazilian Portuguese language to the program.

### CODE REVIEW

**Comment by theRealPadster** [comment]
Thanks for the translation. Before it can be used, it needs to be registered in the locale system. Could you do this step here as well? 
https://github.com/spicetify/marketplace/wiki/Localizing-Marketplace#registering-your-language

**Review by theRealPadster — CHANGES_REQUESTED** [review]
The file also needs a .json extension

**Comment by Kyo-70** [comment]
> Thanks for the translation. Before it can be used, it needs to be registered in the locale system. Could you do this step here as well? https://github.com/spicetify/marketplace/wiki/Localizing-Marketplace#registering-your-language

I've just made the change and I hope it's correct.

**Comment by Kyo-70** [comment]
I'll apologize because I'm not able to integrate another file for change I'm new to the github tools... if you can help me how it's done or just include and edit I'd appreciate it

**Comment by Kyo-70** [comment]
Now I've done it - update file index.ts

File: src/resources/locales/index.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import ru from "./ru.json";
 import uk from "./uk.json";
 import zhCN from "./zh-CN.json";
 import zhTW from "./zh-TW.json";
+import ptBR from "./pt-BR.json";
```
Comment (theRealPadster) [inline src/resources/locales/index.ts:15]: It looks like the linter is complaining that these aren't in alphabetical order 🙄 
Could you put the ptBr one in between "pl" and "ru"?

**Review by theRealPadster — CHANGES_REQUESTED** [review]

**Review by theRealPadster — APPROVED** [review]
Passed the checks finally, thank you!

**Pull request merged**